README.md

```diff
@@ -53,10 +53,10 @@ Here you can find the documentation for different versions:
 - Latest stable version: [GitHub docs](https://github.com/flame-engine/flame/tree/master-v0.x/doc)
 - Latest v1.0.0 version: [GitHub docs](https://github.com/flame-engine/flame/tree/1.0.0-rc8/doc)
 
-The complete documentation can be found [here](doc/README.md).
+The complete documentation can be found [here](https://github.com/flame-engine/flame/doc).
 
-Many examples of different features can be found [here](doc/examples) and a good starting example
-can be found [here](/example).
+We provide many examples of different features which can be tried out from your browser
```

Comment:
why remove the code links? I think we should just add the new link to the list

Reply:
All examples have the link to the code now, so I feel that it is better for us to redirect the user to the live examples, which is way easier do navigate, then when they want to see the code, they can access from there.

But if you think we should have both, let me know and I can put it back

Reply:
But you removed the link for the standalone example as well, that is not included. Also I personally think the code is easier to navigate, but up to you :)

Reply:
I re added and now we have both